22/default.conf

```diff
@@ -55,23 +55,16 @@ server {
             open_file_cache_errors off;
         }
 
-        location ~* /sites/.*/files/advagg_css/ {
+        location ~* /sites/.*/files/optimized/(css|js)/ {
```

Comment:
@Sergey-Orlov maybe that change is for older advagg and 5&6 will work via `files/(css|js|styles)/`

Reply:
well, then I think we don't need separate declaration of location files for advagg

Reply:
Except the case if our image is still used with Drupal7 which could be true